### PR TITLE
Upgrade pillow to >=9.0.0

### PR DESCRIPTION
**Pull request by cristiandean**
Let's do the same as [here](https://github.com/thumbor/thumbor/pull/1390).

### CODE REVIEW

**Comment by marcelometal** [comment]
https://github.com/thumbor/remotecv/commit/bcebf857ff84f2b2275cc84b5eb8d81c8c662985

**Comment by cristiandean** [comment]
✅   Good catch!

**Comment by heynemann** [comment]
It says this PR has conflicts to merge...

File: setup.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
         "remotecv.detectors.profile_detector": ["*.xml"]
     },
     install_requires=[
-        'opencv-python-headless==4.*,>=4.2.0', 'pillow==7.*,>=7.0.0',
+        'opencv-python-headless==4.*,>=4.2.0', 'Pillow>=9.0.0',
```
Comment (heynemann) [inline setup.py:39]:
```suggestion
        'opencv-python-headless==4.*,>=4.2.0', 'Pillow==9.*,>=9.0.0',
```

**Review by heynemann — APPROVED** [review]
Please fix the conflicts that are stopping rebasing and we're good to go. If you feel like it we could remove poetry from this repo as well.

**Comment by cristiandean** [comment]
Do you guys think this is a good idea? If so, I can remove Poetry.

**Comment by heynemann** [comment]
I think we should keep the same in all projects. setup.py only installs.

**Comment by RaphaelVRossi** [comment]
> I think we should keep the same in all projects. setup.py only installs.

Hey @heynemann we merged #43 with only setuptools install.

And we already update Pillow to `>=9.0.0`

**Comment by scorphus** [comment]
Like some other PRs, this too has been superseded by #43. Thanks, @cristiandean.

**Pull request closed without merging**